Removed the complete field item instance referencing Account.Account_Trigger__c; all unrelated FlexiPage metadata and ordering were preserved.

--- before/backup/flexipages/Person_Account.flexipage-meta.xml
+++ after/working/flexipages/Person_Account.flexipage-meta.xml
@@ -57,12 +57,6 @@
                 <identifier>RecordSafe_Number__cField</identifier>
             </fieldInstance>
         </itemInstances>
-        <itemInstances>
-            <fieldInstance>
-                <fieldItem>Record.Account_Trigger__c</fieldItem>
-                <identifier>RecordAccount_Trigger__cField</identifier>
-            </fieldInstance>
-        </itemInstances>
         <name>Facet-bfd1ba90-c017-43e3-ac90-3e4794c312fa</name>
         <type>Facet</type>
     </flexiPageRegions>
